### PR TITLE
Enhancement to Listing page

### PR DESCRIPTION
**Pull request by Txu045**
- added both utility and feature comments to be stored in firebase
- utility/feature comments are added onto listing page
- removed reviews section
- icons added to enhance readability for features and utilities
- additional parameters added into listing details section
- modified cypress test to comply with changes

image

### CODE REVIEW

File: cypress/integration/yosoko/index.spec.js [hunk 1]
```diff
@@ -106,7 +106,6 @@ describe("From index page", () => {
       cy.contains("Lease details");
       cy.contains("Amenities");
       cy.contains("Utilities");
-      cy.contains("Reviews");
```
Comment (meganhng) [inline cypress/integration/yosoko/index.spec.js:109]: What's reviews?

**Review by meganhng — APPROVED** [review]
🚀

File: pages/listings/[id].tsx [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-console */
```
Comment (sttic) [inline pages/listings/[id].tsx:1]: Nitpick: is this needed anymore? looks like it's left over from debugging :stuck_out_tongue:

**Review by sttic — APPROVED** [review]
Looks good! :rocket:

**Pull request merged**